CI fails with flake8 warnings

After adding flake8 check in the CI, this check is fails for several syntax errors in existing files.

```
./falcon/urls.py:23:19: E203 whitespace before ','
./falcon/urls.py:23:20: E231 missing whitespace after ','
./falcon/views.py:3:1: E302 expected 2 blank lines, found 1
./falcon/views.py:4:41: W292 no newline at end of file
```

--- a/falcon/urls.py
+++ b/falcon/urls.py
@@ -20,5 +20,5 @@ from falcon.views import login
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', login ,name='login')
+    path('', login, name='login')
 ]

--- a/falcon/views.py
+++ b/falcon/views.py
@@ -1,4 +1,5 @@
 from django.shortcuts import render
 
+
 def login(request):
     return render(request, 'login.html')

